core/plugins/org.fusesource.ide.camel.model.service.impl/src/org/fusesource/ide/camel/model/service/internal/CamelService.java

```diff
@@ -54,8 +54,9 @@
 	public CamelService() {
 		try {
 			tempFolder = File.createTempFile("grape","m2repo");
-			tempFolder.delete();
-			tempFolder.mkdirs();
+			if(tempFolder.delete()){
```

Comment:
hum in fact, the file has been created the line before... which is really strange, why creating a file to delete and then create a folder?